assets/js/state/actions/cluster.js

```diff
@@ -1,6 +0,0 @@
-export const CHECKS_SELECTED = 'CHECKS_SELECTED';
```

Comment:
I don't understand why you do these changes in this PR :sweat_smile: 
I would create tech-debt tasks and work on them. 
Otherwise the reviews got out of hand.
(And I know that I asked you to not use action files, but you could have only do things belonging to the actual change there)